dsl/src/test/scala/org/http4s/dsl/PathInHttpServiceSpec.scala

```diff
@@ -3,6 +3,7 @@ package dsl
 
 import cats.data.Validated._
 import cats.effect.IO
+import cats.implicits._
```

Comment:
I think `cats.syntax.foldable._` here is preferrable to `.implicits`. I like `.implicits._` for examples more than I do for our own code (and it makes some editors poop themselves with so many syntax objs)

Reply:
I don't like editors that poop themselves, but I'm willing to concede on this.  I still strongly favor it for examples, because new users shouldn't have to pay a huge import tax up front.